Compilation fails with `debug-render-3d` feature disabled

On `main` (currently `ce3847c`),
```toml
bevy_rapier3d = { git = "https://github.com/dimforge/bevy_rapier", rev = "ce3847c", default-features = false, features = [
    "dim3",
    "async-collider",
] }
```
fails to compile with
```
error[E0412]: cannot find type `ColliderDebugColor` in this scope
   --> /home/seldom/.cargo/git/checkouts/bevy_rapier-26af51cff3486e5b/ce3847c/bevy_rapier3d/../src/plugin/plugin.rs:213:30
    |
213 |             .register_type::<ColliderDebugColor>()
    |                              ^^^^^^^^^^^^^^^^^^ not found in this scope
    |
help: you might be missing a type parameter
    |
183 | impl<PhysicsHooks, ColliderDebugColor> Plugin for RapierPhysicsPlugin<PhysicsHooks>
    |                  ++++++++++++++++++++

error[E0412]: cannot find type `ColliderDebugColor` in this scope
   --> /home/seldom/.cargo/git/checkouts/bevy_rapier-26af51cff3486e5b/ce3847c/bevy_rapier3d/../src/plugin/plugin.rs:218:30
    |
218 |             .register_type::<ColliderDebugColor>();
    |                              ^^^^^^^^^^^^^^^^^^ not found in this scope
    |
help: you might be missing a type parameter
    |
183 | impl<PhysicsHooks, ColliderDebugColor> Plugin for RapierPhysicsPlugin<PhysicsHooks>
    |                  ++++++++++++++++++++

For more information about this error, try `rustc --explain E0412`.
error: could not compile `bevy_rapier3d` (lib) due to 2 previous errors
```
but this successfully compiles:
```toml
bevy_rapier3d = { git = "https://github.com/dimforge/bevy_rapier", rev = "ce3847c", default-features = false, features = [
    "dim3",
    "async-collider",
    "debug-render-3d",
] }
```

--- a/bevy_rapier2d/Cargo.toml
+++ b/bevy_rapier2d/Cargo.toml
@@ -46,7 +46,7 @@ wasm-bindgen = ["rapier2d/wasm-bindgen"]
 serde-serialize = ["rapier2d/serde-serialize", "bevy/serialize", "serde"]
 enhanced-determinism = ["rapier2d/enhanced-determinism"]
 headless = []
-async-collider = ["bevy/bevy_asset", "bevy/bevy_scene"]
+async-collider = ["bevy/bevy_asset", "bevy/bevy_scene", "bevy/bevy_render"]
 
 [dependencies]
 bevy = { version = "0.14", default-features = false }

--- a/bevy_rapier3d/Cargo.toml
+++ b/bevy_rapier3d/Cargo.toml
@@ -47,7 +47,7 @@ wasm-bindgen = ["rapier3d/wasm-bindgen"]
 serde-serialize = ["rapier3d/serde-serialize", "bevy/serialize", "serde"]
 enhanced-determinism = ["rapier3d/enhanced-determinism"]
 headless = []
-async-collider = ["bevy/bevy_asset", "bevy/bevy_scene"]
+async-collider = ["bevy/bevy_asset", "bevy/bevy_scene", "bevy/bevy_render"]
 
 [dependencies]
 bevy = { version = "0.14", default-features = false }

--- a/src/plugin/plugin.rs
+++ b/src/plugin/plugin.rs
@@ -210,12 +210,10 @@ where
             .register_type::<ContactSkin>()
             .register_type::<Group>()
             .register_type::<RapierContextEntityLink>()
-            .register_type::<ColliderDebugColor>()
             .register_type::<RapierConfiguration>()
             .register_type::<SimulationToRenderTime>()
             .register_type::<DefaultRapierContext>()
-            .register_type::<RapierContextInitialization>()
-            .register_type::<ColliderDebugColor>();
+            .register_type::<RapierContextInitialization>();
 
         app.insert_resource(Events::<CollisionEvent>::default())
             .insert_resource(Events::<ContactForceEvent>::default())

--- a/src/render/mod.rs
+++ b/src/render/mod.rs
@@ -107,6 +107,7 @@ impl Plugin for RapierDebugRenderPlugin {
     fn build(&self, app: &mut App) {
         app.register_type::<DebugRenderContext>();
         app.register_type::<ColliderDebug>();
+        app.register_type::<ColliderDebugColor>();
 
         app.insert_resource(DebugRenderContext {
             enabled: self.enabled,

